packages/trie/src/trie.ts

```diff
@@ -143,13 +148,15 @@ export class Trie {
   }
 
   static async create(opts?: TrieOpts) {
+    const keccakFunction =
+      opts?.common?.customCrypto.keccak256 ?? opts?.useKeyHashingFunction ?? keccak256
     let key = ROOT_DB_KEY
 
     const encoding =
       opts?.valueEncoding === ValueEncoding.Bytes ? ValueEncoding.Bytes : ValueEncoding.String
 
     if (opts?.useKeyHashing === true) {
-      key = (opts?.useKeyHashingFunction ?? keccak256)(ROOT_DB_KEY) as Uint8Array
+      key = keccakFunction.call(undefined, ROOT_DB_KEY) as Uint8Array
```

Comment:
Not sure, why do we have to `.call` here?

Reply:
:shrug: I think this was left over from when @holgerd77 was trying to introduce a global crypto thing or something.